celesta-sql/src/main/java/ru/curs/celesta/score/Grain.java

```diff
@@ -108,7 +108,11 @@ protected String getErrorMsg(String name) {
         }
 
         modify();
+
         getElementsHolder((Class<T>) element.getClass()).addElement(element);
+        if (element instanceof BasicTable) {
```

Comment:
This `if` doesn't look good and should be supported by tests ))

I didn't find a unit test that checks that e.g a table can be extracted both by requesting a `Table` and a `BasicTable` instance. We should have one!

Reply:
Added to ru.curs.celesta.score.ParserTest.test1()